python/lsst/dc2gen3/patch_dc2.py

```diff
@@ -19,70 +23,210 @@
 # constants
 
 DATATYPES_TO_CONVERT = ("flat", "sky", "SKY")
-POLICY_FILE = Policy.defaultPolicyFile(
-    "obs_lsst", "lsstCamMapper.yaml", "policy"
-)
-FIX_HEADER = False
+POLICY_FILE = Policy.defaultPolicyFile("obs_lsst", "lsstCamMapper.yaml", "policy")
 
 # exception classes
 
 # interface functions
 
 
-def copy_files(origin_repo, new_repo, policy_file=POLICY_FILE):
+def move_files(calib_root, policy_file=POLICY_FILE):
```

Comment:
If these moves are expensive, we really ought to be able to avoid them by changing `ConvertRepoTask`.  If this is already working, that's fine (and easier, from a human perspective), but I wanted to raise that possibility in case there are concerns about doing it at scale.

Reply:
At present, this doesn't seem to be a bottleneck, but I've only been running it on a small test subset of DC2. I'll keep this in mind if it turns out to be a problem when applied to the full repository.